docs/reference/toc.yml

```diff
@@ -1,6 +1,7 @@
 toc:
   - file: index.md
   - file: api-docs.md
-  - file: third-party-dependencies.md
+  - folder: api-reference
+  - folder: third-party-dependencies
```

Comment:
I think that because we're using this folder model rather than a file/children model, these are being listed in a weird order. ideally, we'd list these dependency files from newest to oldest:

<img width="213" height="229" alt="image" src="https://github.com/user-attachments/assets/0982818b-eff4-4ceb-83d0-df086540d822" />

Reply:
yes 😕 , I think it would be a bit more complex to automate the generation of these files with the file/children model.